README.md

```diff
@@ -14,7 +14,7 @@ Below is a listing of plugin versions and respective Velero versions that are co
 
 | Plugin Version  | Velero Version |
 |-----------------|----------------|
-| v1.0.0          | v1.2.0         |
+| v1.0.x          | v1.2.0         |
```

Comment:
This (`v1.0.x`) seemed cleanest to me, but I can list out `v1.0.0` and `v1.0.1` on separate lines if you all prefer.